dazzle.yaml

```diff
@@ -64,20 +64,32 @@ combiner:
       ref:
       - base
       chunks:
-        - lang-node:21
+        - lang-node:22
         - tool-chrome
     - name: node-lts
       ref:
       - base
       chunks:
-        - lang-node:20
+        - lang-node:22
```

Comment:
latest node-lts is 20 https://nodejs.org/en/download/package-manager

Reply:
Nice catch, we'll have to come back to change this in ~ November

![](https://raw.githubusercontent.com/nodejs/Release/main/schedule.svg?sanitize=true)